draft-ietf-rats-msg-wrap.md

```diff
@@ -221,15 +221,16 @@ This always applies, even if the conceptual message format is already textual (e
 When using CBOR, the value field MUST be encoded as a CBOR byte string.
 
 `ind`:
-: An optional bitmap that indicates which conceptual message types are
+: An optional 4-byte bitmap that indicates which conceptual message types are
 carried in the `value` field.  Any combination (i.e., any value between
-1 and 15, included) is allowed.  This is useful only if the `type` is
+1 and 2<sup>32</sup>-1 included) is allowed.  (However, note that only four values are registered at the time of writing, so the acceptable values is currently limited to 1 to 15.)  This is useful only if the `type` is
 potentially ambiguous and there is no further context available to the
 CMW consumer to decide.  For example, this might be the case if the base
 media type is not profiled (e.g., `application/eat+cwt`), if the `value`
 field contains multiple conceptual messages with different types (e.g.,
 both Reference Values and Endorsements within the same `application/signed-corim+cbor`), or if the same profile identifier is
 shared by different conceptual messages.
+The value MUST be non-zero. The absence of conceptual message indicator information is indicated by omitting the `ind` field entirely.
```

Comment:
This is good for deterministic encoding, something I've been spending a lot of time on in the CBOR WG. It avoids two representations at the data model level for the same thing.

Reply:
I knew you’d like this :-)